### ISSUE TITLE
our site footer hides the submit button on the combinedproposal submit page (and likely everywhere)

### ISSUE THREAD

**Issue by tykling** [bornhack/bornhack-website#228]

**Comment by fALKENdk** [comment]
If this is still an issue how do I replicate it?

**Comment by tykling** [comment]
yes it is still an issue, go submit a proposal on a mobile device: https://wwwstaging.bornhack.org/bornhack-2018/program/proposals/submit/talk/

staging has the bootstrap-devsite db loaded so you can login with user1@example.com / user1 to access
